pkgs/by-name/wl/wl-restart/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, scdoc
+, nix-update-script
+}:
+stdenv.mkDerivation rec {
+  pname = "wl-restart";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Ferdi265";
+    repo = "wl-restart";
+    rev = "v0.2.0";
```

Comment:
```suggestion
    rev = "refs/tags/v${finalAttrs.version}";
```

Reply:
i feel like this is a bad idea because the hash doesnt get updated